Packs/FeedCrowdstrikeFalconIntel/Integrations/FeedCrowdstrikeFalconIntel/FeedCrowdstrikeFalconIntel.py

```diff
@@ -0,0 +1,252 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# IMPORTS
+from datetime import datetime
+import requests
+from typing import List, Tuple
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+
+    def __init__(self, params):
+        self._client_id = params.get('client_id')
+        self._client_secret = params.get('client_secret')
+        super().__init__(base_url="https://api.crowdstrike.com/", verify=not params.get('insecure', False),
+                         ok_codes=tuple(), proxy=params.get('proxy', False))
+        self._token = self._get_access_token()
+        self._headers = {'Authorization': 'Bearer ' + self._token}
+
+    @staticmethod
+    def _handle_errors(error_entry: dict) -> str:
+        errors = error_entry.get("errors", [])
+        error_messages = [f"{error['code']}: {error['message']}" for error in errors]
+        error_messages_str = '\n'.join(error_messages)
+        return error_messages_str
+
+    def http_request(self, method, url_suffix, full_url=None, headers=None, json_data=None, params=None, data=None,
+                     files=None, timeout=10, ok_codes=None, return_empty_response=False, auth=None):
+
+        return super()._http_request(method=method, url_suffix=url_suffix, full_url=full_url, headers=headers,
+                                     json_data=json_data, params=params, data=data, files=files, timeout=timeout,
+                                     ok_codes=ok_codes, return_empty_response=return_empty_response, auth=auth)
+
+    def _get_access_token(self) -> str:
+        body = {
+            'client_id': self._client_id,
+            'client_secret': self._client_secret
+        }
+        token_res = self.http_request('POST', '/oauth2/token', data=body, auth=(self._client_id, self._client_secret))
```

Comment:
Doesn't the token have any expiration date?

Reply:
The integration does fetch indicators once every four hours, and the token expires once every half hour, so there is no reason to keep the token.